<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBDevicePageUi</class>
 <widget class="QFrame" name="OpenRGBDevicePageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="DeviceView" name="DeviceViewBox" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="SecondaryLayout">
     <item>
      <layout class="QVBoxLayout" name="ModeAndSelectionLayout">
       <item>
        <layout class="QHBoxLayout" name="ZoneSelectionLayout" stretch="0,2,1">
         <item>
          <widget class="QLabel" name="ZoneLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Zone:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ZoneBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="ResizeButton">
           <property name="text">
            <string>Resize</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="LEDSelectionLayout" stretch="0,2,1">
         <item>
          <widget class="QLabel" name="LEDLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>LED:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="LEDBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="SelectAllLEDsButton">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="ApplyColorsButton">
         <property name="text">
          <string>Apply Colors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SetAllButton">
         <property name="text">
          <string>Set All Devices</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ModeSelectionLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="ModeLabel">
           <property name="text">
            <string>Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ModeBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ColorModeSelectionLayout" stretch="0,1,1,1">
         <item>
          <widget class="QLabel" name="ColorLabel">
           <property name="text">
            <string>Colors:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="PerLEDCheck">
           <property name="text">
            <string>Per-LED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ModeSpecificCheck">
           <property name="text">
            <string>Mode-Specific</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RandomCheck">
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="SpeedLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="SpeedLabel">
           <property name="text">
            <string>Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="SpeedSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="DirectionLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="DirectionLabel">
           <property name="text">
            <string>Dir:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="DirectionBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ColorLayout" stretch="1,0,0,0,0">
       <item>
        <widget class="ColorWheel" name="ColorWheelBox" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="RedAndHueLayout">
         <item>
          <layout class="QHBoxLayout" name="RedLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="RedLabel">
             <property name="text">
              <string>R:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="RedSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="HueLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="HueLabel">
             <property name="text">
              <string>H:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="HueSpinBox">
             <property name="maximum">
              <number>359</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="GreenAndSatLayout" stretch="0,0">
         <item>
          <layout class="QHBoxLayout" name="GreenLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="GreenLabel">
             <property name="text">
              <string>G:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="GreenSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="SaturationLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="SatLabel">
             <property name="text">
              <string>S:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SatSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="BlueAndValLayout">
         <item>
          <layout class="QHBoxLayout" name="BlueLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="BlueLabel">
             <property name="text">
              <string>B:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="BlueSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="ValueLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="ValLabel">
             <property name="text">
              <string>V:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ValSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="QuickColorLayout">
         <item>
          <widget class="QPushButton" name="ButtonRed">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonYellow">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonGreen">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonCyan">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonBlue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonMagenta">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWheel</class>
   <extends>QWidget</extends>
   <header location="global">ColorWheel.h</header>
   <container>1</container>
   <slots>
    <signal>colorChanged(QColor)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>DeviceView</class>
   <extends>QWidget</extends>
   <header location="global">DeviceView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LEDBox</tabstop>
  <tabstop>SetAllButton</tabstop>
  <tabstop>ModeBox</tabstop>
  <tabstop>PerLEDCheck</tabstop>
  <tabstop>ModeSpecificCheck</tabstop>
  <tabstop>RandomCheck</tabstop>
  <tabstop>SpeedSlider</tabstop>
  <tabstop>DirectionBox</tabstop>
  <tabstop>RedSpinBox</tabstop>
  <tabstop>GreenSpinBox</tabstop>
  <tabstop>BlueSpinBox</tabstop>
  <tabstop>HueSpinBox</tabstop>
  <tabstop>SatSpinBox</tabstop>
  <tabstop>ValSpinBox</tabstop>
  <tabstop>ButtonBlue</tabstop>
  <tabstop>ButtonMagenta</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
